<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCChartTP</class>
 <widget class="QWidget" name="QCChartTP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1619</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,10">
     <item>
      <widget class="CusTableView" name="QCProjectsTable" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="QCProductXArea" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color: rgb(191, 191, 191);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="QCProductNameX">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>质控品X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AverageValX">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>均值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="StandardDeviationX">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>标准差</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="QCProductYArea" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color: rgb(191, 191, 191);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="QCProductNameY">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>质控品Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AverageValY">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>均值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="StandardDeviationY">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>标准差</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="QCTipsArea" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color: rgb(191, 191, 191);
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 0);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>历史值</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 170, 127);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>最新值</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,15">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="QCXCheck">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>质控品X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="QCXChecY">
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>质控品Y</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="CusLineChartWidget" name="QCTPChartArea" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="RefreshBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PrintBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>打印</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LJBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>LJ图</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CusTableView</class>
   <extends>QWidget</extends>
   <header location="global">CusTableView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CusLineChartWidget</class>
   <extends>QWidget</extends>
   <header location="global">CusLineChartWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
